raiden/network/rpc/smartcontract_proxy.py

```diff
@@ -42,6 +43,8 @@ def inspect_client_error(val_err: ValueError, eth_node: str) -> ClientErrorInspe
                 return ClientErrorInspectResult.INSUFFICIENT_FUNDS
             elif 'always failing transaction' in error['message']:
                 return ClientErrorInspectResult.ALWAYS_FAIL
+            elif error['message'] == 'replacement transaction underpriced':
+                return ClientErrorInspectResult.TRANSACTION_UNDERPRICED
 
     elif eth_node == EthClient.PARITY:
```

Comment:
Can you somehow find out the corresponding parity error code and handle it there too?